src/app/src/main/kotlin/org/imperial/mrc/hint/controllers/BaselineController.kt

```diff
@@ -16,34 +14,14 @@ class BaselineController(private val fileManager: FileManager,
 
     @PostMapping("/pjnz/")
     @ResponseBody
-    fun uploadPJNZ(@RequestParam("file") file: MultipartFile): String {
-
-        val fileName = file.originalFilename!!
-        fileManager.saveFile(file, FileType.PJNZ)
-
-        // TODO request validation from R API and get back JSON
```

Comment:
Also replaced this TODO from ages ago now that the PJNZ endpoint exists in hintr.